packages/odyssey/CHANGELOG.md

```diff
@@ -47,25 +70,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- bugfix/ee-463 - fixes table cell overflow by wrapping long strings
-- [Docs] Rename "Building Blocks" section to "Foundations"
-
-## [0.2.0] - 2020-04-30
-
-### Added
-- Status: New component and documentation
-- Tab: New component and documentation
-- Tag: New component and documentation
-
-### Changed
-- [Breaking Change] Rename `yellow` color variable to `caution`
-- All focus outline instances now use new outline mixin
-- Adds Status content type to Table component
-- [Docs] Fix misspelling in Form documentation
-- [Docs] Rename "Foundation" to "Base" in nav
-
-## [0.1.3] - 2020-04-10
-
```

Comment:
Why are these entries being removed?

Reply:
These lines are dupe of the surrounding releases.

Reply:
Ah - it was hiding! Thanks!